### PR TITLE
Feature/issue2880

### PR DESCRIPTION
**Pull request by chrisala**

### CODE REVIEW

**Review by temi — COMMENTED** [review]
Wow, a big change. LGTM except for a minor comment.

File: grails-app/views/organisation/_projects.gsp [hunk 1]
```diff
@@ -1,232 +0,0 @@
-<g:if test="${organisation.projects}">
```
[the remark this answers is not in the document]
Comment (chrisala) [reply grails-app/views/organisation/_projects.gsp:1]: This gsp fragement was replaced when the NHT related changes were deployed last year (with /shared/projectListByProgram) to match the management unit page behaviour - I just deleted this now as a tidy up.

[the remark this answers is not in the document]
Comment (chrisala) [reply grails-app/views/organisation/_projects.gsp:1]: Thanks @temi yes, I've fix this.

**Review by temi — APPROVED** [review]

**Comment by temi** [comment]
@chrisala build seems to have failed.

**Comment by chrisala** [comment]
> @chrisala build seems to have failed.

This is fixed now (the test caught the extra view model only field in the payload and there was also a test ordering dependency that got exposed).

**Pull request merged**